core/services/pipeline/task.eth_tx.go

```diff
@@ -159,8 +154,11 @@ func (t *ETHTxTask) Run(ctx context.Context, lggr logger.Logger, vars Vars, inpu
 		SignalCallback:   true,
 	}
 
-	if minOutgoingConfirmations > 0 {
-		// Store the task run ID, so we can resume the pipeline when tx is confirmed
+	if !isMinConfirmationSet {
+		// Store the task run ID, so we can resume the pipeline when tx is finalized
+		txRequest.PipelineTaskRunID = &t.uuid
```

Comment:
@jmank88 is `PipelineTaskRunID` only set during this specific functionality? I'm wondering if there is a case where it's being set for a different reason and the `minConfirmations` gets executed as a side-effect of that.

Reply:
> is PipelineTaskRunID only set during this specific functionality?

Yes

> and the minConfirmations gets executed as a side-effect of that.

What do you mean that it "gets executed"? AFAIK this parameter only affects the callback behavior, not confirmation.

Reply:
I'm referring to a scenario where `PipelineTaskRunID` is used by another task/process, but it seems it doesn't.